src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -1972,6 +1973,8 @@ namespace HeatBalanceManager {
         GetZoneData(state, ErrorsFound); // Read Zone data from input file
 
         SurfaceGeometry::SetupZoneGeometry(state, ErrorsFound);
+
+        ZoneEquipmentManager::GetZoneEquipment(state);
```

Comment:
Now called from GetBuildingData. And GetBuildingData is called early in the simulation inside GetHeatBalanceInput which calls several setup functions.

    void GetHeatBalanceInput(EnergyPlusData &state)
    {
        GetProjectControlData(state, ErrorsFound);

        GetSiteAtmosphereData(state, ErrorsFound);

        GetWindowGlassSpectralData(state, ErrorsFound);

        Material::GetMaterialData(state, ErrorsFound); // Read materials from input file/transfer from legacy data structure

        GetFrameAndDividerData(state);

        GetConstructData(state, ErrorsFound); // Read constructs from input file/transfer from legacy data structure

        GetBuildingData(state, ErrorsFound); // Read building data from input file

        DataSurfaces::GetVariableAbsorptanceSurfaceList(state);

        GetIncidentSolarMultiplier(state, ErrorsFound);

        // Added SV 6/26/2013 to load scheduled surface gains
        GetScheduledSurfaceGains(state, ErrorsFound);

        if (state.dataSurface->UseRepresentativeSurfaceCalculations) {